kornia/augmentation/_2d/geometric/affine.py

```diff
@@ -91,7 +91,7 @@ def __init__(
         return_transform: Optional[bool] = None,
     ) -> None:
         super().__init__(p=p, return_transform=return_transform, same_on_batch=same_on_batch, keepdim=keepdim)
-        self._param_generator = cast(rg.AffineGenerator, rg.AffineGenerator(degrees, translate, scale, shear))
+        self._param_generator: rg.AffineGenerator = rg.AffineGenerator(degrees, translate, scale, shear)
```

Comment:
if doesn't work here without the cast even ?

Reply:
maybe the `_param_generator` is not well typed in the parent class? @shijianjian